RecoTracker/LSTCore/standalone/README.md

```diff
@@ -106,186 +187,56 @@ Comparing two different runs
-```
-
-### Inclusion of LST in other CMSSW packages
-Including the line
-```
-<use name="lst"/>
-```
-in the relevant package `BuildFile.xml` allows for
-including our headers in the code of that package.
+    runTheMatrix.py -w upgrade -n -e -l 24834.704
 
-## Running LST in a CVMFS-less setup
-
-The setup scripts included in this repository assume that the [CernVM File System (CVMFS)](https://cernvm.cern.ch/fs/) is installed. This provides a convenient way to fetch the required dependencies, but it is not necessary to run LST in standalone mode. Here, we briefly describe how to build and run it when CVMFS is not available.
-
-The necessary dependencies are CUDA, ROOT, the Boost libraries, Alpaka, and some CMSSW headers. CUDA, ROOT, and Boost, are fairly standard libraries and are available from multiple package managers. For the remaining necessary headers you will need to clone the [Alpaka](https://github.com/alpaka-group/alpaka) and [CMSSW](https://github.com/cms-sw/cmssw) repositories. The Alpaka repository is reasonably sized, but the CMSSW one extremely large, especially considering that we only need a tiny fraction of its files to build LST. We can get only the Alpaka interface headers from CMSSW by running the following commands.
-
-``` bash
-git clone --filter=blob:none --no-checkout --depth 1 --sparse --branch CMSSW_14_1_X https://github.com/cms-sw/cmssw.git
-cd cmssw
-git sparse-checkout add HeterogeneousCore/AlpakaInterface
-git checkout
-```
+For convenience, the workflow has been run for 100 events and the output is stored here:
 
-Then all that is left to do is set some environment variables. We give an example of how to do this in lnx7188/cgpu-1.
+    /data2/segmentlinking/CMSSW_14_1_0_pre0/step2_21034.1_100Events.root
```

Comment:
is this used in the CI?
I recall that we've updated to something more fresh than 21034

Reply:
Good catch! It's 29834.1